Quickstart.module.scss

```diff
@@ -154,4 +157,53 @@
         color: #0e0e0e;
       }
     }
+  }
+  .copy_button {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    color: #c2c2c2;
```

Comment:
Perhaps you can make a new global css variable for this in index.scss, in the :root.